week-1-ts-ecosystem/w01e01-typescript-basics/src/types/index.ts

```diff
@@ -0,0 +1,11 @@
+export interface Address {
+  street: string;
+  no: number;
+}
+
+export interface User {
+  name: string;
+  age: number;
+  email: string;
+  address: Address;
+}
```

Comment:
ten katalog zamiast `types` nazwałbym `models` bo to Twój model danych `User` i też tak można nazwać plik zamiast `index.ts` to `User.ts` - żeby od razu było widać jaki masz model danych w plikacji.